lib/classes/plugin_manager.php

```diff
@@ -1940,7 +1940,7 @@ public static function standard_plugins_list($type) {
             ),
 
             'qbank' => array(
-                 ''
+                 'viewquestiontype',
```

Comment:
I realise that this line has an extra space (5 instead of 4) but this came from MDL-71516 so probably when we merge the plugins there we can fix it.